cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/TestCommand.java

```diff
@@ -274,7 +274,7 @@ public String getName() {
 
     @Override
     public void printLongDesc(StringBuilder out) {
-        out.append("Compiles Ballerina modules and create balo files. \n");
+        out.append("compile Ballerina modules and create .balo files. \n");
```

Comment:
Is this description even relevant for the test command?

Reply:
Changed to "test the modules in a Ballerina program".